cpp/memilio/utils/custom_index_array.h

```diff
@@ -563,30 +602,54 @@ class CustomIndexArray
     }
 
     /**
-     * @brief slice creates a slice into the multidimensional array
+     * @brief slice creates a slice into the multidimensional array.
```

Comment:
In general: Is Creates lowercase? When using a period at the end of the sentence it looks weird not having a uppercase letter, but i guess it is style

Reply:
Some conventions apply of course. I don't think we have these written down anywhere and we haven't been consistent here.

I would suggest for readability, it probably should start with a capital letter. Documentation for functions should also start with verb, functions do something. Naming the function or class in the brief doc string is unnecessary.